.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
> That's what the .pylintrc file is for. I would like to eliminate any per-file configuration.

Ah yes, definitely (sorry if it seemed like I was suggesting to use per-file configuration).